### ISSUE TITLE
LOSS IS NEGATIVE

### ISSUE THREAD

**Issue by tulasiram58827** [noahchalifour/rnnt-speech-recognition#8]
What does it mean if the loss is negative. Do I have to wait more or is there anything wrong?

Attaching the required reference. 

 1/1561 [..............................] - ETA: 5:59:49 - loss: -0.00392020-02-05 15:46:10.353673: E tensorflow/core/profiler/internal/gpu/cupti_tracer.cc:1329] function cupti_interface_->EnableCallback( 0 , subscriber_, CUPTI_CB_DOMAIN_DRIVER_API, cbid)failed with error CUPTI_ERROR_INVALID_PARAMETER
2020-02-05 15:46:10.353737: I tensorflow/core/profiler/internal/gpu/device_tracer.cc:88]  GpuTracer has collected 0 callback api events and 0 activity events.
 957/1561 [=================>............] - ETA: 9:32 - loss: -0.1399

**Comment by noahchalifour** [comment]
@tulasiram58827 Did you install the `warp-transducer` loss? If not you should get an output log that says `RNN-T Loss function not found.` That is the most likely reason. Also are you running on CPU or GPU?

**Comment by tulasiram58827** [comment]
@noahchalifour Yes I missed installing warp-transducer loss. Now I installed it and I am getting the following error.
tensorflow.python.eager.core._SymbolicException: Inputs to eager execution function cannot be Keras symbolic tensors, but found [<tf.Tensor 'spec_lengths:0' shape=(None,) dtype=int32>, <tf.Tensor 'label_lengths:0' shape=(None,) dtype=int32>]

Can u look into it

**Comment by noahchalifour** [comment]
Pulling the latest commit will fix this issue, error has to do with multi GPU support #6 .
